src/animations/backwardsCompatibleAnimWrapper.js

```diff
@@ -0,0 +1,69 @@
+import {
+  always,
+  block,
+  call,
+  clockRunning,
+  cond,
+  set,
+  startClock,
+  stopClock,
+} from '../base';
+import { default as Clock } from '../core/AnimatedClock';
+import { default as Value } from '../core/AnimatedValue';
+
+function backwardsCompatibleInvoke(node, AnimationClass, value, config) {
+  let returnMethod;
+  const newValue = new Value(0);
+  const newClock = new Clock();
+  const currentState = AnimationClass.getDefaultState();
+  currentState.position = newValue;
+  let alwaysNode;
+  let isStarted = false;
+  return {
+    start: currentReturnMethod => {
+      if (isStarted) {
+        returnMethod && returnMethod({ finished: false });
+        return;
+      }
+      isStarted = true;
+      alwaysNode = always(
+        set(
+          value,
+          block([
+            cond(currentState.finished, set(currentState.finished, 0)),
```

Comment:
are you sure about this one? What's the purpose of `finished` if we just always set it to 0.

I think we should only expect `finished` to be updated by the animation node

Reply:
I made it to be the same like in RN, but as it's not crucial, gonna drop it